src/lib/nodes/btc/types/api/common.ts

```diff
@@ -23,6 +23,7 @@ type Method =
   | 'getaddressbalance'
 
 export type RpcRequest<P = any, M extends Method | string = string> = {
+  baseURL?: string
```

Comment:
Remove it. Axios already contains this property.

![image](https://github.com/user-attachments/assets/9098b431-3089-474a-9682-c1d2e49fe6fb)